### PR TITLE
Options Panel Completed.

### PR DESCRIPTION
**Pull request by CSellers720**
### Added blurbs at bottom of panel
blurb
### Here's a shot of the whole panel
options-panel

### CODE REVIEW

File: client/src/components/ProductOptions/components/DropHint.test.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```
Comment (ccali11) [inline client/src/components/ProductOptions/components/DropHint.test.jsx:1]: This commit was just rearranging your components / file structure, yes?

**Review by ccali11 — COMMENTED** [review]
Looks good.

File: client/src/components/ProductOptions/ProductOptions.jsx [hunk 2]
```diff
@@ -99,7 +113,12 @@ const ProductOptions = (props) => {
         <ChooseSetting>CHOOSE THIS SETTING</ChooseSetting>
         <AddToWishlist selected={wishClick} onClick={e => setWishClick(!wishClick)}><Heart selected={wishClick}>❤</Heart> WISH LIST</AddToWishlist>
         <ShippingBlurb />
-        <DropHint>Drop Hint</DropHint>
+        <DropHint>
+          <HeartEnvelope src='https://radiant-dearth.s3.us-east-2.amazonaws.com/icons/drop-hint.png' />
```
[the remark this answers is not in the document]
Comment (CSellers720) [reply client/src/components/ProductOptions/ProductOptions.jsx]: I downloaded a vector image and edited it in GIMP to get the colors I wanted, as well as adding an alpha layer (which allows for a transparent background). I just googled 'heart envelope' and picked one I liked.

File: client/src/components/ProductOptions/components/DropHint.test.jsx (hunk 1 above)
Comment (CSellers720) [reply client/src/components/ProductOptions/components/DropHint.test.jsx:1]: This was just me creating a base test environment that I can come back and fill out in the future. I've kind of let TDD fall by the wayside in favor of speed, but I plan to circle back and implement testing. This is basically a reminder that I need to fill a test in here.

**Comment by CSellers720** [comment]
Merged in CLI

**Pull request closed without merging**